### ISSUE TITLE
Suggestion - bike for burgershot delivery

### ISSUE THREAD

**Issue by sakar5157** [InZidiuZ/legacy-assets-issue-tracker#741]
### Name

BlackHawk#3322

### Summary

As there is only one van as a delivery vehicle. it would be better if there are options available on the purpose. Bikes are more realistic in case of delivering the single orders as we have to pass through alleys for the purpose.

### Link

I couldnt find the link. I have seen similar police bikes . Anything with a livery can work.

### Reason

players are requesting for the option for faster and realistic RP

**Comment by MetaHatter** [comment]
Only concern I would have is potentially throwing off job payout balance by letting people deliver faster by doing so without getting off the bike.  

Would be really weird if doordash showed up to my house sitting on a motorcycle on my front porch.

**Comment by sakar5157** [comment]
> Only concern I would have is potentially throwing off job payout balance by letting people deliver faster by doing so without getting off the bike.
> 
> Would be really weird if doordash showed up to my house sitting on a motorcycle on my front porch.

I dont think we can directly finish the delivery rn without getting off  the vehicle. If it is like that, its better to make it obvious to get out of bike to finish delivery.

**Comment by seamarob** [comment]
> 

Would using a scooter help? It’d be a slower arrival time in the case of people not getting off vehicles.

**Comment by InZidiuZ** [comment]
This issue tracker is being shut down and replaced by Discord threads. If this issue is still relevant, create a thread on [Discord](https://discord.gg/opfw).
